### PR TITLE
Add support of a specific port for the tests

### PR DESCRIPTION
**Pull request by matrixise**

### CODE REVIEW

File: certificate_watcher.py [hunk 1]
```diff
@@ -12,13 +12,14 @@
 from datetime import datetime, timedelta
 import socket
 import ssl
-import sys
 
 
 __version__ = "0.0.2"
 
+TLS_PORT = 443
```
Comment (JulienPalard) [inline certificate_watcher.py:19]: Maybe pedantic but I'd name it HTTPS_PORT.

Comment (matrixise) [reply certificate_watcher.py:19]: IMAPS is 993, and it's a TLS port.

**Comment by JulienPalard** [comment]
Thanks for the PR Stéphane!

**Comment by JulienPalard** [comment]
Released it as 0.0.4 on pypi.

**Pull request merged**